account_broker/model/account_invoice.py

```diff
@@ -0,0 +1,132 @@
+# -*- encoding: utf-8 -*-
+###########################################################################
+#    Module Writen to OpenERP, Open Source Management Solution
+#
+#    Copyright (c) 2013 Vauxoo - http://www.vauxoo.com/
+#    All Rights Reserved.
+#    info Vauxoo (info@vauxoo.com)
+############################################################################
+#    Coded by: Luis Ernesto García Medina (ernesto_gm@vauxoo.com)
+############################################################################
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class account_invoice_tax(models.Model):
+    _inherit = 'account.invoice.tax'
+
+    tax_partner_id = fields.Many2one('res.partner', 'Supplier', readonly=True)
+
+
+class AccountInvoiceLine(models.Model):
+    _inherit = 'account.invoice.line'
+
+    invoice_broker_id = fields.Many2one(
+        'account.invoice', 'Invoice Broker', help='If this invoice line is to '
+        'the payment to a broker, indicate the original invoice that must be '
+        'generated the broker partner', domain=[('type', '=', 'in_invoice')])
+    partner_broker_ok = fields.Boolean(
```

Comment:
Hi @moylop260 ,
I attach the images with the error that is occasioned by your proposals

With ``` <field name="invoice_broker_id" attrs="{'invisible':[('parent.partner_broker_ok','=',False)]}"/>```
http://awesomescreenshot.com/0ae4efirae
With ```<field name="invoice_broker_id" attrs="{'invisible':[('parent.partner_id.partner_broker_ok','=',False)]}"/>```
http://awesomescreenshot.com/0cc4efiwd7

It´s by this that I used the field function in the lines, have a comment about this?

Regards